File: 面向对象程序设计（Java）/实验二 类与对象/立方体 (Cube class).java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        for (int i = 1; i <= t; i++) {
            double edge = sc.nextDouble();
            Cube obj = new Cube(edge);
            obj.calArea();
            obj.calVolume();
            System.out.println("obj" + i + " " + obj);

        }
    }


}

class Cube {
    private double edge;
    private double volume;
    private double area;

    public Cube(double edge) {
        this.edge = edge;
    }

    void calVolume() {
        this.volume = edge * edge * edge;
    }

    void calArea() {
        this.area = 6 * edge * edge;
    }

    @Override
    public String toString() {
        return String.format("Volume:%.2f; Area:%.2f", this.volume, this.area);
    }

    void display() {
        System.out.println(this.toString());
    }
}

File: python应用开发/CSOJ/实验05-面向对象编程/立方体 (Cube class).py
class Cube:
    def __init__(self, edge):
        self.edge = edge
        self.volume = 0
        self.area = 0

    def cal_volume(self):
        self.volume = self.edge ** 3

    def cal_area(self):
        self.area = 6 * (self.edge ** 2)

    def __str__(self):
        return f"Volume:{self.volume:.2f}; Area:{self.area:.2f}"

    def display(self):
        print(self.__str__())


if __name__ == "__main__":
    t = int(input())
    for i in range(1, t + 1):
        edge = float(input())
        obj = Cube(edge)
        obj.cal_volume()
        obj.cal_area()
        print(f"obj{i} {obj}")